github-pullrequest-plugin/src/main/java/org/jenkinsci/plugins/github/pullrequest/dsl/GitHubPRJobDslExtension.java

```diff
@@ -44,8 +45,20 @@ public Object commitStatusOnGH(Runnable closure) {
                 context.unstableAs(),
                 null,
                 null,
-                null
-        );
+                null);
```

Comment:
if you have some formatter i can place it in repo. IDEA by default messes only imports with hashes